site/features/reactor.html

```diff
@@ -27,23 +27,35 @@ <h1>FusionAuth Reactor</h1>
         </p>
       </div>
       <div class="col-xs-12 col-lg-4 feature-card">
         <img src="/assets/img/icons/advanced-registration-form.svg" alt="Advanced Registration Forms">
-        <h2>Advanced Registration Forms</h2><span class="blue stamp">Coming soon!</span>
+        <h2>Advanced Registration Forms</h2><span class="orange stamp">NEW!</span>
+        <p>
+          You can build custom forms and perform complex validation during registration without writing any code. Our Advanced Registration Forms provides you the ability to collect custom user data during registration and store it within FusionAuth. 
+        </p>
+        <h4>Build Multi-Page Custom Forms</h4>
         <p>
-          Soon you'll be able to collect custom user data and perform complex validation during registration inside FusionAuth. Our Advanced Registration Forms will provide the ability to collect custom user data and store it within FusionAuth. Admins will be able to edit this user data inside FusionAuth's admin UI as well.
+          Use standard form fields like email, name, and password, or create your own. Custom fields are typed and can be strings, numbers, dates or booleans. You can also create multi-page registration forms, collecting different information on each page.
         </p>
       </div>
       <div class="col-xs-12 col-lg-4 feature-card">
-        <img src="/assets/img/icons/enhanced-threat-detection.svg" alt="Enhanced Thread Detection">
-        <h2>Enhanced Threat Detection</h2><span class="blue stamp">Coming soon!</span>
+        <img src="/assets/img/icons/database-gear.svg" alt="LDAP Integration">
+        <h2>LDAP Integration</h2><span class="orange stamp">NEW!</span>
+        <p>
+          Want to connect external data sources such as LDAP to FusionAuth? Now you can with Connectors. When you configure a connector, specify whether it is only for a certain email domain or all domains. You can also configure user data migration rules on a per-connector basis. 
+        </p>
+        <h4>Authenticate? Migrate? Why Not Both?</h4>
         <p>
-          Worried that hackers might be attacking your application? FusionAuth will perform threat detection each time a user logs in. Using an adaptive threat model based on numerous factors, FusionAuth will determine if an authentication looks like a high risk event. When FusionAuth detects an elevated risk, you'll be able to force a multi-factor authentication or lock the user's account automatically.
+          When a user provides an email address matching a domain, FusionAuth will authenticate them against the external provider, allowing that system to be the system of record. If configured, user information will be migrated from the external system into FusionAuth, which allows for a staged transition from legacy systems.
```

Comment:
`allowing that system to be the system of record` - too many uses of `system` in one sentence?

Reply:
The domain setting is optional. Really what we need to say here is that the rules to how users are authenticated and migrated is completely customizable.

Reply:
OK, I reworked this.